src/pages/communication-page/communication.ts

```diff
@@ -52,21 +52,24 @@ interface CommunicationPageState {
 })
 export class CommunicationPage extends BasePageComponent {
 
-  readonly providedEmail = 'providedEmail';
-  readonly newEmail = 'newEmail';
+  readonly providedEmail = 'Provided';
```

Comment:
If you declare these variables as static, they become constants and won't be changeable even from the constructor.